src/components/PrivacyPolicy/PrivacyPolicyPage.tsx

```diff
@@ -0,0 +1,228 @@
+'use client';
+
+import { PageContainer } from '@/components/Containers/PageContainer';
+import { DynamicPagesContainer } from '@/components/DynamicPagesContainer';
+import { Box, Typography, List, ListItem, ListItemText } from '@mui/material';
+import { format } from 'date-fns';
+
+export const PrivacyPolicyPage = () => {
+  const currentDate = format(new Date(), 'MMMM dd, yyyy');
+
+  const sections = [
```

Comment:
Should we move all this content inside the translation file and then call the `t(<KEY_HERE>)`?